lib/puppetx/simp/compliance_mapper.rb

```diff
@@ -374,135 +374,181 @@ def import(filename, data)
         def list_puppet_params(profile_list)
```

Comment:
This method is 174 lines long.

Reply:
Yup. Should be fixed at some point but this was changing as little code as possible.